src/api/python/CMakeLists.txt

```
@@ -19,6 +19,35 @@ if(POLICY CMP0057)
   cmake_policy(SET CMP0057 NEW)
 endif()
 
+# Check that scikit-build is installed
+# Provides CMake files for Python bindings
+check_python_module("skbuild" "scikit-build")
+
+# setuptools for installing the module
+check_python_module("setuptools")
+
+# Find cmake modules distributed with scikit-build
+# They are distributed under <scikit-build directory>/resources/cmake
+execute_process(
+  COMMAND
+    ${PYTHON_EXECUTABLE} -c "from __future__ import print_function;\
+                             import os; import skbuild;\
+                             print(os.path.dirname(skbuild.__file__) + \
+                             os.path.sep + 'resources' + os.path.sep + 'cmake', end='')"
```

```suggestion
                             print(os.path.join(os.path.dirname(skbuild.__file__), 'resources', 'cmake'), end='')"
```
Probably needs formatting.

---

What exactly do you want to check with this? If `<scikit-build directory>/resources/cmake` exists? If yes, wouldn't it be better toi use `os.path.exists` instead?

---

The point is only to get the path to the cmake modules. But it makes sense to check that the directory exists, especially given the error message below. Added with 4123e353d578a6e712396f8c808f7aba8b99b6b0.